src/schema/users.ts

```diff
@@ -1630,6 +1635,30 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         },
       );
     },
+    userIntegration: async (
```

Comment:
We do fetch all integrations either way for dropdown so we can just maybe search inside `useIntegrationsQuery` or is it used somewhere where `useIntegrationsQuery` is not already triggered?

Reply:
Yeah it's used quicker and singular actually.
So create squad with slack -> enter `new squad` page and here we need this singular integration only.

Reply:
👍

Reply:
Then only if you can add a test for this query please